Makefile

```diff
@@ -1,11 +1,20 @@
 TAG := $(shell git log --format=%h -1)
-IMAGE ?= taccaci/hazmapper:$(TAG)
+
+.PHONY: build-angular
+build-angular:
+	docker build -t taccaci/hazmapper:$(TAG) -f angular/Dockerfile .
+	docker tag taccaci/hazmapper:$(TAG) taccaci/hazmapper:latest
+
+.PHONY: build-react
+build-react:
+	docker build -t taccaci/hazmapper-react:$(TAG) -f react/Dockerfile .
+	docker tag taccaci/hazmapper-react:$(TAG) taccaci/hazmapper-react:latest
 
 .PHONY: build
 build:
-	docker build -t $(IMAGE) -f angular/Dockerfile .
-	docker tag taccaci/hazmapper:${TAG} taccaci/hazmapper:latest
+	make build-angular && make build-react
 
 .PHONY: deploy
 deploy:
```

Comment:
Shouldn't this target be named `publish` to distinguish it from the actual `deploy` step?

Reply:
Yes, makes sense 👍